src/BufferCopy/ColorBufferToRDRAM.cpp

```diff
@@ -177,25 +177,14 @@ bool ColorBufferToRDRAM::_prepareCopy(u32 _startAddress)
 	}
 
 	if (m_pCurFrameBuffer->m_scale != 1.0f) {
-		u32 x0 = 0;
-		u32 width;
-		if (config.frameBufferEmulation.nativeResFactor == 0) {
-			const u32 screenWidth = wnd.getWidth();
-			width = screenWidth;
-			if (wnd.isAdjustScreen()) {
-				width = static_cast<u32>(screenWidth*wnd.getAdjustScale());
-				x0 = (screenWidth - width) / 2;
-			}
-		} else {
-			width = m_pCurFrameBuffer->m_pTexture->realWidth;
-		}
+		u32 width = m_pCurFrameBuffer->m_pTexture->realWidth;
```

Comment:
You killed read from frame buffer for widescreen-hack mode.
We can't just read from frame buffer while in widescreen mode because our wide buffer does not fit its place in RDRAM.
We need to crop part of the image, which corresponds to original 4:3 frame.
The code you removed does that.

Reply:
Ok, I figured there was some scenerio I was missing, I still need to think of some way to do this properly